src/isoConstants/iso3166-1.ts

```diff
@@ -0,0 +1,260 @@
+// taken from https://github.com/umpirsky/country-list/blob/master/data/en/country.json
+// Locally, we do not check this file as the local Typescript server in editors cannot handle the large JSON files.
+// On CI, we will remove this full comment block and will still typecheck this file
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
+export const COUNTRY_LOOKUP = {
```

Comment:
linh, you can use `makeEnum` and use type overrides to get this to work as an enum as well https://github.com/transcend-io/main/blob/d7737f8881c2d22ab4c8c21be702ce1fdaaaee24/backend-support/aws-scripts/src/enums.ts#L18

Reply:
nvm i see you already do this in the `isoRegion.ts` file -- just make sure to include the type overrides as well!

Reply:
From my understanding of the convo [here](https://github.com/transcend-io/main/pull/17237#discussion_r942758707), we're splitting region into two columns, one for country and other subdivision, so it's alright just to export the two individual enums? Is there still any need to have the intersection between the two enums? I can just remove if not needed anymore

Reply:
hm sure - let's split them but make sure that whenever you use `makeEnum` that you also do the type override (linked above) so that we have them as enums as not just as an object